src/WaitingRoom.js

```diff
@@ -11,7 +12,10 @@ const cardStyle = {
 export default class WaitingRoom extends Component {
   constructor(props) {
     super(props);
-    this.state = { sessions: null };
+    this.state = {
+        sessions: null,
+        filterType: "all",
```

Comment:
Let's use the same enum style as you did for `DB_STATE`:

```
const FILTER_TYPE = Object.freeze({
  ALL: 0,
  MY_GAMES: 1,
  ...
}
```

You can export this variable as well so it's accessible in `FilterButton.js`.

Reply:
Indeed